CHANGELOG.md

```diff
@@ -1,3 +1,243 @@
+### citus v11.0.0_beta (March 22, 2022) ###
+
+* Turns MX on by default
+
+* Adds `citus_finalize_upgrade_to_citus11()` which is necessary to upgrade to
+  Citus 11+ from earlier versions
+
+* Distributes functions with `CREATE FUNCTION` command
+
+* `citus_shard_indexes_on_worker` shows all local shard indexes regardless of
+  `search_path`
+
+* `citus_shards_on_worker` shows all local shards regardless of `search_path`
+
+* Adds `citus.create_object_propagation` GUC to control ddl creation behaviour
+  in transactions
+
+* Sets `create_object_propagation` default to `immediate`
+
+* Uses a fixed `application_name` while connecting to remote nodes
+
+* Hides shards based on `application_name` prefix
+
+* Prevents `citus.node_conninfo` to use `application_name`
+
+* Starts identifying rebalancer backends by `application_name=citus_rebalancer`
+
+* Starts internal backends by `application_name=citus_internal`
+
+* Adds `citus.enable_unsafe_triggers` flag to propagate triggers on distributed
+  tables
+
+* Adds `citus.max_client_connections` GUC to limit non-Citus connections
+
+* Adds `fix_partition_shard_index_names` udf to fix currently broken names
+
+* Adds propagation for foreign server commands
+
+* Adds propagation of `CREATE AGGREGATE` commands
+
+* Add support for `ALTER FUNCTION ... SUPPORT ...` commands
+
+* Adds support for `CREATE SCHEMA AUTHORIZATION` statements without schema name
+
+* Adds support for `TRUNCATE` for foreign tables
+
+* Adds support for adding partitioned local table to Citus metadata
+
+* Adds support for automatic binary encoding in re-partition joins when possible
+
+* Adds support for foreign tables in MX
+
+* Adds support for operator class parameters in indexes
+
+* Allows `create_distributed_function()` on a function owned by an extension
+
+* Allows creating distributed tables in sequential mode
+
+* Allows disabling nodes when multiple failures happen
+
+* Creates type locally if it has undistributable dependency
+
+* Delegates function calls of the form `SELECT .. FROM func()`
+
+* Deprecates `master_get_table_metadata` UDF
+
+* Deprecates Inactive shard state, never marks any placement inactive
+
+* Disables auto-undistributing for local tables added by the user
+
+* Disables distributed & reference foreign tables
+
+* Disallows remote execution from queries on shards
+
+* Drops `citus_worker_stat_activity`
+
+* Drops `citus.enable_cte_inlining` GUC
+
+* Drops `citus.single_shard_commit_protocol` GUC, defaults to 2PC
+
+* Drops support for `citus.multi_shard_commit_protocol`, always use 2PC
+
+* Enables re-partition joins in transaction blocks
+
+* Enables re-partition joins on followers
+
+* Enables shard replication > 1 hash distributed tables on Citus MX
+
+* Errors out if object has circular dependency
+
+* Errors out if object has dependency to an object with unsupported type
```

Comment:
since we still allow creating them locally ?

```suggestion
* Prevents propagating objects having a dependency to an object with unsupported type
```